core/app/WebSocketServer.hs

```diff
@@ -56,20 +56,21 @@ broadcast conn msg = do
   T.putStrLn ("Log:" <> cs msg)
   WS.sendTextData conn msg
 
-application :: State -> WS.ServerApp
-application state pending = do
+application :: FilePath -> Notebook -> State -> WS.ServerApp
+application fp nb state pending = do
```

Comment:
For instance, here, I'm absolutely in favor of short names, as the type already denotes what they are 👍 😁